released/packages/coq-pi-agm/coq-pi-agm.1.2.7/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+maintainer: "yves.bertot@inria.fr"
+
+homepage: "http://www-sop.inria.fr/members/Yves.Bertot/"
+bug-reports: "yves.bertot@inria.fr"
+license: "CECILL-B"
+build: [["coq_makefile" "-f" "_CoqProject" "-o" "Makefile" ]
+       [ make "-j" "%{jobs}%" ]]
+install: [ make "install" "DEST='%{lib}%/coq/user-contrib/pi_agm'" ]
+depends: [
+  "ocaml"
+  "coq" {<= "8.17" & < "8.20~"}
```

Comment:
```suggestion
  "coq" {>= "8.17" & < "8.20~"}
```
Surely this should not be any version at or below 8.17? It should be between 8.17 and 8.19, right?